### PR TITLE
Run3-alca212B Remove compilation warnings in the clases of EcalAlCaRecoProducers|EcalCalibAlgos within Calibration

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Remove compilation warnings in the clases of EcalAlCaRecoProducers|EcalCalibAlgos within Calibration

#### PR validation:

Use the runTheMatrix test workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/EcalAlCaRecoProducers/src/AlCaElectronsTest.h [hunk 1]
```diff
@@ -23,266 +26,171 @@
...
-   m_endcapGlobalCrystalsEnergy->Write () ;
-   m_endcapLocalCrystalsEnergy->Write () ;   
-   m_barrelGlobalCrystalsEnergyMap->Write () ;
-   m_endcapGlobalCrystalsEnergyMap->Write () ;
-   output.Close () ;
-   //PG save root things
-   return ;
-}
-
+void AlCaElectronsTest::endJob() {}
```
Comment (tvami) [inline Calibration/EcalAlCaRecoProducers/src/AlCaElectronsTest.h:112]: If this is not doing anything, could be removed, right?

Comment (bsunanda) [reply Calibration/EcalAlCaRecoProducers/src/AlCaElectronsTest.h]: Done

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by tvami** [comment]
+alca
 * code changes in line w PR describtion
 * Jenkins tests pass

**Comment by qliphy** [comment]
+1

**Pull request merged**